### PR TITLE
Adding foundations to root

### PR DESCRIPTION
**Pull request by patrickhuie19**

### CODE REVIEW

**Review by krehermann — APPROVED** [review]

**Review by krehermann — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # 1. Per Github docs: "Order is important; the last matching pattern takes the most precedence."
 # Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
 
+# Root
+/* @smartcontractkit/foundations
+
```
Comment (ilija42) [inline CODEOWNERS:7]: does this override` /core @smartcontractkit/foundations` ?

**Review by jmank88 — DISMISSED** [review]

**Review by krehermann — DISMISSED** [review]

**Review by ilija42 — DISMISSED** [review]

**Review by jmank88 — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # 1. Per Github docs: "Order is important; the last matching pattern takes the most precedence."
 # Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
 
-# Core
-/core @smartcontractkit/foundations
+# Root
+/* @smartcontractkit/foundations
```
Comment (chainchad) [inline .github/CODEOWNERS]: Is the goal to have all subdirs too? If so you'd want `/**`.

**Review by jmank88 — DISMISSED** [review]

**Review by ilija42 — DISMISSED** [review]

**Review by eutopian — APPROVED** [review]

File: .github/CODEOWNERS
Comment (jmank88) [inline .github/CODEOWNERS:1]: Oh, not it is invalid. Did we lose some teams?

**Review by jmank88 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (eutopian) [reply .github/CODEOWNERS:1]: If you don't count the ones we lost yesterday briefly :D

**Pull request merged**